app/services/catalogue_services_client.rb

```diff
@@ -96,10 +92,25 @@ def create_request(requester:, request:)
     raise ItemRequestError.new("Failed to request item (#{request[:item_id]}) for requester (#{requester})")
   end
 
-  def request_limit_reached?(requester:)
-    conn = Faraday.new(url: ENV["CATALOGUE_SERVICES_API_BASE_URL"]) do |f|
-      f.response :json
+  def request_details(request_id:)
+    conn = setup_connection
+
+    res = conn.get("/catalogue-services/folio/request/#{request_id}")
```

Comment:
You'll also need to pass me the loan=<true|false> parameter from the "loan" property I send you in the request summary. This tells me which Folio endpoint I have to use the get the details.

Reply:
Ah, I saw it this morning and couldn't remember if I was supposed to do something about that. Thanks, will make the change.

Reply:
@scott-yeadon I've made the change to pass the "loan" param to the endpoint.